modules/custom/module2/module2.module

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Implements hook_theme().
+ *
+ * Register a module or theme's theme implementations.
```

Comment:
No need to write this entire thing
 * Register a module or theme's theme implementations.
 * The implementations declared by this hook specify how a
 * particular render array is to be rendered as HTML.
 
 Only  `Implements hook_theme().` is enough